pages/docs/tutorials/native/basic-kotlin-native-app.md

```diff
@@ -2,90 +2,275 @@
+the Language & Frameworks | Kotlin section.
+
+
+At this point, we should have a Gradle project that is ready to be opened in an IDE.
+IntelliJ IDEA (CLion, AppCode, or AndroidStudio) helps us to generate the
+[Gradle Wrapper](https://docs.gradle.org/current/userguide/gradle_wrapper.html)
+scripts for our project. 
+
+Now let's open the project in IntelliJ IDEA. For that we click on the File | Open... and select
+our 
+<span class="multi-language-span" data-lang="kotlin">
+`build.gradle.kts` 
+</span><span class="multi-language-span" data-lang="groovy">
+`build.gradle`
+</span>
+project file. 
+
+![Open Project Dialog]({{ url_for('tutorial_img', filename='native/basic-kotlin-native/idea-open-as-project.png')}}){: width="70%"}
+
+Confirm to open the file _as Project_.
+
+![Gradle Import Dialog]({{ url_for('tutorial_img', filename='native/basic-kotlin-native/idea-import-gradle.png')}})
+
+Select _Use gradle 'wrapper' task configuration_ option in the Gradle import dialog to complete the import.
+For existing projects, which already have Gradle wrapper scripts, the _Use default Gradle wrapper_
+option should be selected instead.
+
+Use the path to the Java runtime version 1.8 or 11 for the _Gradle JVM_ field. Check out the 
+[https://jdk.java.net/11](https://jdk.java.net/11/) or [https://adoptopenjdk.net/](https://adoptopenjdk.net/)
+for the best JRE, OpenJDK, or JDK distribution.  
 
 ## Running the application
 
-To run the application, we can merely invoke it
+Usually, a native binary can be compiled as _debug_ with more debug information and fewer optimizations and _release_,
+where optimizations are enabled, and there is no (or at least less) debug information available.  
+The binary files are created in the `build/bin/native/debugExecutable` or `build/bin/native/releaseExecutable`
+folders respectively. The file has a `.kexe` extension on Linux and macOS and an `.exe` extension on Windows. Use the following command
+to instruct the build to produce binaries:
+
+<div class="multi-language-sample" data-os="linux">
+<div class="sample" markdown="1" theme="idea" mode='bash' data-highlight-only>
+
+```bash
+./gradlew build
+```
+</div>
+</div>
+
+<div class="multi-language-sample" data-os="macos">
+<div class="sample" markdown="1" theme="idea" mode='bash' data-highlight-only>
 
 ```bash
-./program.kexe
+./gradlew build
 ```
+</div>
+</div>
 
-It's important to understand that this is now a native application, and no runtime or virtual machine is required. The output should be
+<div class="multi-language-sample" data-os="windows">
+<div class="sample" markdown="1" theme="idea" mode='bash' data-highlight-only>
 
 ```bash
+gradlew.bat build
+```
+</div>
+</div>
+
+It's important to understand that this is now a native application, and no
+runtime or virtual machine is required.
+We can now run the compiled binary from the console:
+
+```
+> Task :runDebugExecutableNative
```

Comment:
Placing this output here confuses because the way to get it is not described by this point.